kanidmd/src/lib/core/https.rs

```diff
@@ -1310,17 +1311,22 @@ pub fn create_https_server(
     );
 
     // Add routes
-    tserver.at("/").get(index_view);
-    tserver
-        .at("/pkg")
-        .serve_dir(env!("KANIDM_WEB_UI_PKG_PATH"))
-        .map_err(|e| {
-            error!(
-                "Failed to serve pkg dir {} -> {:?}",
-                env!("KANIDM_WEB_UI_PKG_PATH"),
-                e
-            );
-        })?;
+
+    // If we are no-ui, we remove this.
+    if !matches!(role, ServerRole::WriteReplicaNoUI) {
```

Comment:
Can this be replaced with a ServerRole method? It might be premature opt but also... maybe not?

```
- if !matches!(role, ServerRole::WriteReplicaNoUI)
+ if role.should_show_ui() {
```

```
impl ServerRole {
     fn should_show_ui(&self) -> bool {
         !matches(self, WriteReplicaNoUI)
     }
 }
 ```

Reply:
I think it's premature opt for now. When we need it, we can cross that bridge I think :)